vigetlabs/wordpress-site-starter
# [#95] Fixing button outline border bug

# Summary
This fixes a few bugs with the button. 

- Option to change the bg color and text color.
- The outline variant defaults to a transparent bg.
- When you set the text color it should set the border color.

## Issues

* #95
## Testing Instructions

1. Add a block that has a button or just add the button block. 
2. Make sure you can change the bg/text color.
3. The Outline variant defaults to a transparent bg.
4. When you set the text color for the outline variant it sets the border color.

## Screenshots

Can set a bg/text color overriding the default for both variants. 
<img width="239" alt="Screenshot 2024-05-29 at 12 28 30 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/bf842079-9d31-4067-98e7-0cd3ec2bae36">

When you set the text color it should set the border color.
<img width="187" alt="Screenshot 2024-05-29 at 12 30 48 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/c328330e-243f-46b8-aacb-c7d716da056f">
